raiden-ts/scripts/copyContracts.js

```diff
@@ -1,4 +1,4 @@
-const fs = require('fs');
```

Comment:
I guess we need to add this to our dependencies, even thought it's a transitive dep already!?

Reply:
Let's avoid adding a new dependency for something so simple.. [here](https://stackoverflow.com/a/52338335/1895161) is some code which we can use to copy the whole contracts dir to the `dist` folders